napari/layers/points/points.py

```diff
@@ -923,161 +860,167 @@ def shown(self):
 
     @shown.setter
     def shown(self, shown):
-        self._shown = np.broadcast_to(shown, self.data.shape[0]).astype(bool)
+        self._shown = np.broadcast_to(shown, len(self._points_data)).astype(
+            bool
+        )
         self.refresh()
 
     @property
-    def edge_width(self) -> np.ndarray:
-        """(N, D) array: edge_width of all N points."""
-        return self._edge_width
+    def border_width(self) -> np.ndarray:
```

Comment:
also for each old properties there need to be deprecated property implemented (calling internally the new property).

Reply:
fixed, thanks @Czaki